### ISSUE TITLE
Possible typo WebClient docs

### ISSUE THREAD

**Issue by fadelgaber** [spring-projects/spring-boot#21568]
On [this page](https://docs.spring.io/spring-boot/docs/2.0.3.RELEASE/reference/html/boot-features-webclient.html):

```java
@Service
public class MyService {

	private final WebClient webClient;

	public MyService(WebClient.Builder webClientBuilder) {
		this.webClient = webClientBuilder.baseUrl("http://example.org").build();
	}

	public Mono<Details> someRestCall(String name) {
		return this.webClient.get().url("/{name}/details", name)
						.retrieve().bodyToMono(Details.class);
	}

}
```
`this.webClient.get().url("/{name}/details", name)` should be `uri` instead?

**Comment by philwebb** [comment]
Good spot! Thanks for reporting the issue.

**Comment by wilkinsona** [comment]
This is a duplicate of https://github.com/spring-projects/spring-boot/pull/14499. It has already been fixed in 2.0.6 and later.
